docs/Deploying-Pega-on-openshift.md

```diff
@@ -265,6 +267,7 @@ Configure the following parameters so the pega.yaml Helm chart matches your depl
     | docker.registry.url: username: password: | Include the URL of your Docker registry along with the registry “username” and “password” credentials. | <ul><li>url: “\<URL of your registry>” </li><li>username: "\<Registry account username\>"</li><li> password: "\<Registry account password\>"</li></ul> |
     | docker.pega.image:       | Specify the Pega-provided `Pega` image you downloaded and pushed to your Docker registry.  | Image: "<Registry host name:Port\>/my-pega:\<Pega Platform version>" |
     | tier.name: ”web” tier.ingress.domain:| Set a host name for the pega-web service of the DNS zone. Pega supports specifying certificates for an ingress using the same methods that OpenShift supports. Note that if you configure both secrets and pre-shared certificates on the ingress, the load balancer ignores the secrets and uses the list of pre-shared certificates. For details, see [Using multiple SSL certificates in HTTP(s) load balancing with Ingress](https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-multi-ssl).  | <ul><li>tier.name: "\<the host name for your web service tier\>" </li><li>Assign this host name with an external IP address and log into Pega Platform with this host name in the URL. Your web tier host name must comply with your networking standards and be available as an external IP address.</li><li>tier.ingress.tls: set to `true` to support HTTPS in the ingress. See step 12 to support the management of the certificates in your deployment.</li></ul> |
+    | tier.name: ”web” tier.service.tls:| Set this flag as `true` if the traffic between the loadbalancer/ingress and pods needs to be encrypted. Pega ships with a default self-signed certificate and also supports specifying your own certificate.  | <ul><li>tier.service.tls.enabled: set to `true` to enable the traffic encryption </li><li>tier.service.tls.port: 443</li><li>tier.service.tls.targetPort: 8443</li><li>tier.service.tls.keystore: The base64 encoded content of the keystore file, leaving it empty will take a default self-signed certificate</li><li>tier.service.tls.keystorepassword: the password of the keystore file</li><li>tier.service.tls.cacertificate: the base64 encrypted content of the destination CA certificate. This should be the CA certificate chain for the certificate which is present in the keystore.</li><li>tier.service.traefik.enabled: set to `false` as this option is for a different provider</li></ul> |
```

Comment:
| tier.name: ”web” tier.service.tls:| Set this parameter as `true` to encrypt the traffic between the load balancer/ingress and pods. Beginning with Helm version _____ Pega provides a default self-signed certificate; Pega also supports specifying your own CA certificate.  | <ul><li>tier.service.tls.enabled: set to `true` to enable the traffic encryption.</li><li>tier.service.tls.port: 443</li><li>tier.service.tls.targetPort: 8443</li><li>tier.service.tls.keystore: The base64 encoded content of the keystore file. Leave this value empty to use the default, Pega-provided self-signed certificate.</li><li>tier.service.tls.keystorepassword: the password of the keystore file</li><li>tier.service.tls.cacertificate: the base64 encrypted content of the destination CA certificate. Use this parameter to specify the CA certificate chain for the certificate which is present in the keystore. </li><li>tier.service.traefik.enabled: set to `false` as this option is for a different provider</li></ul> |

Not clear on this still>>tier.service.traefik.enabled: set to `false` as this option is for a different provider.</li>

You should probably add some clarity about why you can't leave this value empty for openshift...it's not clear, but it seems important to not leave blank>>>tier.service.tls.cacertificate:

Reply:
added additional description